docs/spec.md

```diff
@@ -1724,27 +1731,30 @@ when `comparison_direction` is `GE`, `GT`, `LE` or `LT`
 
 | Label | Name                   | Type                                                    | Constraints |
 |-------|------------------------|---------------------------------------------------------|-------------|
-| (I1)  | `lhs`                  | tensor                                                  | (C1-C3)     |
-| (I2)  | `rhs`                  | tensor                                                  | (C1), (C2)  |
+| (I1)  | `lhs`                  | tensor or per-tensor quantized tensor                   | (C1-C4)     |
+| (I2)  | `rhs`                  | tensor or per-tensor quantized tensor                   | (C1-C2)     |
 | (I3)  | `comparison_direction` | enum of `EQ`, `NE`, `GE`, `GT`, `LE`, and `LT`          |             |
 | (I4)  | `compare_type`         | enum of `FLOAT`, `TOTALORDER`, `SIGNED`, and `UNSIGNED` | (C3)        |
 
 #### Outputs
 
-| Name     | Type                   | Constraints |
-|----------|------------------------|-------------|
-| `result` | tensor of boolean type | (C2)        |
+| Name     | Type                                                | Constraints |
+|----------|-----------------------------------------------------|-------------|
+| `result` | tensor of boolean type or per-tensor quantized type | (C2)        |
 
 #### Constraints
 
-* (C1) `element_type(lhs) = element_type(rhs)`.
-* (C2) `shape(lhs) = shape(rhs) = shape(result)`.
+* (C1) `type(lhs) = type(rhs)`.
+* (C2) `shape(lhs) = shape(result)`.
```

Comment:
How about we keep the existing C1/C2 (replacing element_type with baseline_element_type) to keep this PR focused and defer this change to a follow-up PR?